wormhole-connect/src/config/index.ts

```diff
@@ -19,7 +19,13 @@ const el = document.getElementById('wormhole-connect');
 if (!el)
   throw new Error('must specify an anchor element with id wormhole-connect');
 const configJson = el.getAttribute('config');
-export const config: WormholeConnectConfig = JSON.parse(configJson!) || {};
+export const config: WormholeConnectConfig = JSON.parse(configJson!) || {
+  searchTx: {
```

Comment:
I think we want to remove this, because when we open up the netlify preview it brings us to the Resume transaction page

Reply:
Explicitly requesting changes on this. Similar to my comment on the other PR, I don't think we want to be changing the default config for everyone like this.

Reply:
Perhaps it was a left over from testing and was comitted without intention

Reply:
yes, this needs to be removed before we merge to development, it was for local testing and some integration test that I'm trying to archive, I'll remove it once I'm done

Reply:
removed